src/components/pages/doc.js

```diff
@@ -179,13 +181,20 @@ class Doc extends React.Component {
       </div>
     );
 
-    const customAlerts = [];
+    const alerts = [];
     const headerActions = [];
 
+    if (isProfileInsufficient(user)) {
+      alerts.push({
+        message: <InsufficientProfileWarning />,
+        type: ALERT_TYPE.info
+      });
```

Comment:
Since this is repeated selectively on some pages, maybe we can have an alerts module that gets the array of alerts by passing to it all it needs (`user`, `revision` etc.) plus the list of alerts types it wants to get for the current page (`['insufficient-profile', 'archived-document']).
Or by an object with properties being the alert type and values being the required data:
```
getAlerts({ insufficientProfile: { user }, archived: { revision } });
```
This would also insure the same priority/sorting of alerts.

Does that sound ok or an overkill? For a separate ticket of course.

Reply:
We do have the insufficient profile warning an mostly all pages, so that one could be centralized, but most other alerts will probably be very specific to a single page, so I'd prefer to keep them in the page. Maybe we could do this only for the "global" alerts?

Reply:
True, or then keep it simple then and extract an `alerts` catalog-type module with all alerts e.g.
```
const ALERT: {
   name: {
        message: ...,
        type: ...
   }
```
?

Reply:
Should I do something regarding this discussion on this MR?

Reply:
If you feel it's easy and can do it here, yes, if not let's make a ticket.

Reply:
It's at least not hard :-) Then I will extract the global alerts (i.e. insufficient profile) to a separate module.

Reply:
I've extracted the global alerts into a module, please re-check.